### DISCUSSION TITLE
Anyone having issues loading the latest AIO in Safari?

### DISCUSSION THREAD

**Discussion by sol8712** [nextcloud/all-in-one#4871] [Questions] [answered]
I cant seem to get Safari to load my aio instance, ive had multiple iphone users report this, and testing in a web based Safari for Windows emulator shows a blank page. No issues with Chome or Firefox browsers.

Is this an isolated case?

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this at https://help.nextcloud.com/. Thanks!
